chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/SchedulerCertifiedProductSearchDetailsAsync.java

```diff
@@ -33,7 +34,7 @@
             final Long id, final CertifiedProductDetailsManager certifiedProductDetailsManager)
                 throws EntityRetrievalException {
         CertifiedProductSearchDetails dto = certifiedProductDetailsManager.getCertifiedProductDetails(id);
-        getLogger().info("Finishing Details for: " + id);
+        getLogger().info("{}Finishing Details for: {}", this.prefix, id);
```

Comment:
Do you want a space after the first {} ?

Reply:
Nope. If "this.prefix" is "" I don't want to add an unused space. I put a space in the this.prefix for each specific job